metricbeat/mb/mb.go

```diff
@@ -84,6 +90,42 @@ type EventsFetcher interface {
 	Fetch() ([]common.MapStr, error)
 }
 
+// Reporter is used by a MetricSet to report events, errors, or errors with
+// metadata. The methods return false iff publishing failed because the
```

Comment:
lol, makes sense now. Did not make the connection when reading it and just thought: typo :-)